code/game/gamemodes/dynamic/dynamic_rulesets_midround.dm

```diff
@@ -742,8 +745,8 @@
 	required_candidates = 0
 	weight = 3
 	cost = 8
-	minimum_players = 27
-	repeatable = TRUE
+	minimum_players = 32
+	repeatable = FALSE
```

Comment:
Undocumented changes to spiders. Sneaky.

Reply:
I suck at making PRs dont judge :( I'll document it